tests/e2e/performance/test_pod_reattachtime.py

```diff
@@ -165,6 +165,8 @@ def test_pod_reattach_time_performance(
         node_two = worker_nodes_list[1]
 
         time_measures = []
+        files_written_list = []
+        data_written_list = []
```

Comment:
Change the list declaration to something like in this line https://github.com/red-hat-storage/ocs-ci/blob/85eb8969f04638ab8c55a749dd85ab70f80abbf1/ocs_ci/ocs/scale_lib.py#L127
 i.e. it can be `time_measures, files_written_list, data_written_list = ([], [], [])`

Reply:
Sure.